gotom-api-doc.yaml

```diff
@@ -929,6 +929,10 @@ components:
                 description: Share of the platform in percent (42.5% = 42.5).
                 example: 42.5
                 type: number
+              note:
+                description: Subline note for the Platform.
+                example: Subline Remarks Test
```

Comment:
Hier würde ich auf das Wort "Test" verzichten.
Eher vll. "Remark for this FlightSubline" oder so.